### PR TITLE
Multi shop view support

### PR DESCRIPTION
**Pull request by alexbridge**
## Description

Multi shop view support

## Type of change

- [ ] Bug fix
- [X] New feature
- [ ] This change requires a documentation update

### CODE REVIEW

**Review by Carsten89 — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 ### Fixed
 - updateSession pipeline was not called every time when session changed. Caused emtpy cart in engage.
 
+## [2.10.0] - 2020-09-10
```
Comment (Rekhyt) [inline CHANGELOG.md]: Move to "Unreleased".

```suggestion
```

File: README.md [hunk 2]
```diff
@@ -12,6 +12,20 @@ Download and unzip the [latest releases](https://github.com/shopgate/cart-integr
 
 You can find more information in our [support center](https://support.shopgate.com/hc/en-us/articles/202798446-Connecting-to-Shopware) and our [developer documentation](https://docs.shopgate.com/).
 
+## Shop multi views and translations
```
Comment (Rekhyt) [inline README.md:15]:
```suggestion
## Shop Multi Views and Translations
```

File: README.md [hunk 3]
```diff
@@ -12,6 +12,20 @@ Download and unzip the [latest releases](https://github.com/shopgate/cart-integr
...
+
+We need distinguish url of plugin location to allow Shopware load correct shop view data, locale and configuration
```
Comment (Rekhyt) [inline README.md:17]:
```suggestion
We need distinguishable URLs for the plugin location to enable Shopware loading the correct shop view data, locale and configuration.
```

File: src/Backend/SgateShopgatePlugin/Models/Export/Product.php [hunk 4]
```diff
@@ -814,8 +885,9 @@ public function prepareBasePrice($price, $detail)
 
             $basePrice .= " / {$referenceUnit} {$unit}";
         } elseif ($this->getPurchaseSteps($detail) > 1) {
+            $label = $detail->getUnit()->getName();
```
Comment (Rekhyt) [inline src/Backend/SgateShopgatePlugin/Models/Export/Product.php:888]: Looks like `$detail->getUnit()` could be returning null in this branch of the `if` statement. Shouldn't we keep the fall back to "pro Stück" here?

File: README.md [hunk 5]
```diff
@@ -12,6 +12,20 @@ Download and unzip the [latest releases](https://github.com/shopgate/cart-integr
...
+
+Shop settings generally should be
+
+- `virtual Url` field point to (example)
+    - /shop/de for DE shop
+    - /shop/en for EN shop
+
+Plugin access uri should be accordingly:
+- /shop/de/shopgate/plugin for DE shop view and translations
+- /shop/en/shopgate/plugin for EN shop view and translations
```
Comment (Rekhyt) [inline README.md:27]:
```suggestion
Example:

If the shop is set up to have its virtual URL point to
- `/shop/de` for the German shop
- `/shop/en` for the English shop

then access to the plugin should be
- `/shop/de/shopgate/plugin` to get contents (catalog etc.) in German
- `/shop/en/shopgate/plugin` to get contents (catalog etc.) in English
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -8,6 +8,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 ### Fixed
 - updateSession pipeline was not called every time when session changed. Caused emtpy cart in engage.
 
+## [2.9.92] - 2020-09-10
```
Comment (Rekhyt) [inline CHANGELOG.md:11]:
```suggestion
```

**Review by Rekhyt — CHANGES_REQUESTED** [review]

**Review by Shopgate-Travis — APPROVED** [review]

**Review by Rekhyt — APPROVED** [review]

**Pull request merged**